.env

```diff
@@ -0,0 +1,16 @@
+DB_HOST=localhost
```

Comment:
Create a separate `.env` file for test? Production credentials are never to be stored in source code.